File: dataStructure/bubbleSort/review.js
const bubbleSort = (arrayOfNumbers = []) => {
  for (let index = 0; index < arrayOfNumbers.length; index++) {
    let sorted = true;
    for (let j = 0; j < arrayOfNumbers.length -1; j++) {
      if (arrayOfNumbers[j] > arrayOfNumbers[j + 1]) {
        sorted = false;
        let temp = arrayOfNumbers[j];
        arrayOfNumbers[j] = arrayOfNumbers[j + 1];
        arrayOfNumbers[j + 1] = temp;
      }
    }
    if (sorted) {
      break;
    }
  }
  return arrayOfNumbers;
};

console.log(bubbleSort([5,9,10,3,-1]))

File: dataStructure/bubbleSort/review.py
def bubble_sort(list_of_numbers: list) -> list:
    for i in range(len(list_of_numbers)):
        sorted = True
        for j in range(len(list_of_numbers)-1):
            if list_of_numbers[j] > list_of_numbers[j + 1]:
              sorted = False
              temp = list_of_numbers[j]
              list_of_numbers[j] = list_of_numbers[j + 1]
              list_of_numbers[j + 1] = temp
        if sorted:
            break
    return list_of_numbers


print(bubble_sort([5,9,10,3,-1]))